src/diff.c

```diff
@@ -1863,6 +1872,16 @@ int git_diff_commit_as_email(
 	opts.summary = git_commit_summary(commit);
 	opts.author = git_commit_author(commit);
 
+	/* The first line of the commit message is identical to
+	 * the summary and thus needs to be skipped */
+	if ((ptr = git_commit_message(commit)) != NULL) {
+		/* Find end of first line */
+		ptr = strpbrk(ptr, "\r\n");
```

Comment:
Right you are.  In that case, why are we looking for `\r`?  Does Git actually allow `\r` as a line terminator?  Or is the goal to find the `\n` but back up over the `\r` if the line was terminated with `\r\n`?